pkg/mark/markdown_test.go

```diff
@@ -61,3 +61,18 @@ func TestExtractDocumentLeadingH1(t *testing.T) {
 
 	assert.Equal(t, "a", actual)
 }
+
+func TestRemoveNewlinesWithinParagraphTags(t *testing.T) {
+	html := `<p>This newline
+	should be ignored</p>
+	<span>And this one
+	shouldn't</span>`
+
+	expected := `<p>This newline should be ignored</p>
+	<span>And this one
```

Comment:
Is this actually what is expected to be a result? Should not it be like `this newline\tshould be ignored`? As far as I see, the main idea was to remove only linebreaks.

Reply:
As far as I'm aware, the goal to to avoid weird breaks in the confluence behavior; that involves flowing text in `<p>`, so `\t` should be removed as well

Reply:
We'd like this feature as well but I agree that it should only replace linebreaks and not all space. The regex is incorrect.

What's the status of this? It hasn't had updates in a while so looks stale. Wasn't sure if I should put up another PR